### ISSUE TITLE
How to edit tables JSON files ?

### ISSUE THREAD

**Issue by tum34-dot** [dozingcat/Vector-Pinball#64]
Hello,

I am trying to edit some tables to remove the `"retractWhenHit": true` attribute. I would like the ball savers to behave as walls and never retract.

I am not a programmer. So I tried the quick and dirty way: I unzipped the apk and tried to change the JSON files to 
 -`"retractWhenHit": false`
 -no "retractWhenHit" attribute.

But then I can't install the modified apk.

I must be missing something. Could you suggest another way to do this ?

**Comment by dozingcat** [comment]
Yeah, I would expect there to be errors like signature mismatches if you just try to modify the files in the APK directly. Unfortunately the the best way is probably to install Android Studio and build the app yourself.

I've occasionally thought of having a way to load new or modified tables from within the app. The problem is that while the layout is just JSON, the table logic is in Java code which is not convenient to load dynamically.

**Comment by tum34-dot** [comment]
Thank you.
Signature mismatches would prevent the installation. I might try building the app one day...

That other part is also interesting. Having an extra folder where the app would search for extra tables would be great and enable user customization. Do you mean that Java code prevents dynamic loading ?
